package.json

```diff
@@ -31,7 +32,9 @@
 		"eslint": "9.22.0",
 		"husky": "9.1.7",
 		"jetpack-cli": "workspace:*",
-		"jetpack-js-tools": "workspace:*"
+		"jetpack-js-tools": "workspace:*",
+		"stylelint": "16.16.0",
+		"stylelint-scss": "6.11.1"
```

Comment:
Is this actually required?

If so, is it still required if we do something like [this](https://github.com/Automattic/jetpack/blob/2824c7433abb295e4bd97b338fcf2328cfa3fd61/.pnpmfile.cjs#L29-L38) to work around [WordPress/gutenberg#69684](https://href.li/?https://github.com/WordPress/gutenberg/issues/69684) until that's fixed?

Reply:
Indeed, it seems to work with it removed now: 4e0eac7123